kafka-server/Makefile

```diff
@@ -1,8 +1,23 @@
+KAFKA_EMBEDDED_ENV_VERSION ?= 3.2.7
+KAFKA_EMBEDDED_ENV_COMMIT ?= a5a21c8c6593d30860160cff54ffd350f7e1a0d5
 
 usage:        ## Show this help
 	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
 
+patch:        ## Clone and patch the kafka-embedded-env repo, build code into local Maven repo
+	test -e kafka-embedded-env || git clone https://github.com/navikt/kafka-embedded-env
+	(cd kafka-embedded-env; git checkout $(KAFKA_EMBEDDED_ENV_COMMIT))
+	# note: using perl instead of sed below, for better Linux/MacOS cross-compatibility
+	perl -i -pe 's|<jvmTarget>17<|<jvmTarget>11<|g' kafka-embedded-env/pom.xml
+	perl -i -pe 's|<maven\.compiler\.source>17<|<maven.compiler.source>11<|g' kafka-embedded-env/pom.xml
+	perl -i -pe 's|<maven\.compiler\.target>17<|<maven.compiler.target>11<|g' kafka-embedded-env/pom.xml
```

Comment:
FYI: We could just set properties with the command line option `-D` in maven, f.e.
```
mvn -Dmaven.compiler.source=11 -Dmaven.compiler.target=11 install
```
Unfortunately this only works for user-defined properties, so the patch for jvmTarget (as a non-parameterized plugin config) would still be necessary :(
For a future upgrade, we could maybe file a simple PR in the repo to use the `maven.compiler.target` for this value instead of hard-coding it...